docs/_documentations/overview.md

```diff
@@ -1,29 +1,83 @@
+
+2. **Ability to add your own standardised project templates** Create your own [project templates](mdt-vsc-workingwithtemplates.html) and your development teams can access them within their IDE. 
+
+3. **Ability to add your own applications** Access the seamless container development and deployment experience on your IDE that Codewind offers for your current projects.
+
+4. **Auto-build-and-run on code changes** As you develop, Codewind automatically pushes your code changes to your container as efficiently as possible. 
+
+5. **Ability to toggle between auto-build and manual build settings** Have control over when you want to implement Codewind's auto-build feature.
+
+6. **Project details displayed in a user-friendly panel on your IDE** Use Codewind's Project Overview interface to instantly view the endpoint, container ID, location on disk, build and run status, and ports.
+
+7. **Access application, build and container logs and run commands from within the container without leaving your IDE** Codewind integrates with your IDE's terminal window and PROBLEMS and OUTPUT panels for a smooth test and debug experience. Additionally, you can open a shell session *inside* your container with a simple click.
+
+8. **Support for your IDE's debug features**<sup>*</sup> Full compatibility with the debug functionality of your IDE.
+
+9. **Visualize your container application's footprint such as CPU and memory usage** <sup>*</sup> Codewind's Application Metrics Dashboard exposes your microservice's core metrics, including HTTP(S) request and throughput information, as well as CPU usage, memory footprint, and heap.
+
+10. **Integrated load-testing** <sup>*</sup> Configure and run load tests against your microservices using [Codewind's performance dashboard](https://www.youtube.com/watch?v=nfJt3f5TUvc). Use the graphs to compare your repeated load test results and iterate on performance improvements. 
+
+<sup>*</sup> where supported.
+
+### What Integrated Development Environments (IDEs) does Codewind support?
+
+Codewind is available as a desktop IDE extension for [**VS Code**](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) and [**Eclipse**](https://marketplace.eclipse.org/content/codewind), and as a cloud-based IDE extension for [**Eclipse Che**](https://www.eclipse.org/codewind/mdt-che-installinfo.html).
+
+### What cloud native technology does Codewind support?
+Codewind supports [Kubernetes](https://kubernetes.io/) and [OpenShift](https://www.openshift.com/) for container orchestration. 
+
+### Where do I develop my code? Where can I build and run my containers? How does Codewind support my development flow?
+Codewind supports different configurations for developing your containerised applications, so you can use Codewind with whichever development flow you choose - you can develop containerized applications on your desktop, directly in the cloud, or a combination of both:
```

Comment:
American spelling: "containerized" (The first instance of this word needs editing...the second is good. :) )